strings-csharp/ReplaceSpecialCharactersInString/ReplaceSpecialCharactersInString/StringReplacer.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
+
+namespace ReplaceSpecialCharactersInString
+{
+    public class StringReplacer
+    {
+        public static string ReplaceUsingStringReplace(string originalString,
+            char oldChar, char newChar)
+        {
+            return originalString.Replace(oldChar, newChar);
+        }
+
+
+        public static string ReplaceUsingStringBuilder(string originalString,
+            char oldChar, char newChar)
+        {
+            var sb = new StringBuilder(originalString);
+
+            sb.Replace(oldChar, newChar);
+
+            return sb.ToString();
+        }
+
+
+        public static string ReplaceUsingRegex(string originalString,
+            string oldChar, string newChar)
```

Comment:
All methods should have same datatype "char" for old and new character. If required convert char to string within function.